roles/openshift_storage_nfs_lvm/defaults/main.yml

```diff
@@ -6,6 +6,10 @@ osnl_nfs_export_options: "*(rw,sync,all_squash)"
 # mounted as <osnl_mount_dir>/test1g0001 etc.
 osnl_mount_dir: /exports/openshift
 
+# Mode for the NFS-exported directories. Defaults to secure mode which may require additional configs depending on the environment.
+# See https://docs.openshift.com/container-platform/3.5/install_config/persistent_storage/persistent_storage_nfs.html#nfs-volume-security
+osnl_exported_dir_mode: 0700
```

Comment:
Makes sense. I updated the commit with the new variable naming convention. Is this documented somewhere?
Happy to look into refactoring the whole role for consistency. However this would break users configs if they are using the current/old variable names. I'm thinking about updating the variables names and use set_fact to set them based on the old variables values if those are found/defined. WDYT?